docs/tutorials/enterprise_beginner/3_use_existing_blocks.md

```diff
@@ -238,153 +215,37 @@ The Block is already operational, but first update the stylesheet. Add the follo
-            {% for idx in 0..2 %}
-                <div data-studio-slot>
-                    {% if items[idx] is defined %}
-                        {{ render(controller('ez_content:viewLocation', {
-                            'locationId': items[idx],
-                            'viewType': 'list'
-                        })) }}
-                    {% endif %}
-                </div>
-            {% endfor %}
-        </div>
-    </div>
-{% endspaceless %}
-```
-
-You also need a template for the list view for Articles (`app/Resources/views/list/article.html.twig`):
-
-``` html
-<div class="other-article">
-    <div class="other-article-image">
-        {{ ez_render_field(content, 'image', {
-            'parameters': {
-                'alias': 'other_article'
-             }
-        }) }}
-    </div>
-    <h5>
-        <a class="other-article-link" href="{{ path('ez_urlalias', {'contentId': content.id}) }}">{{ ez_content_name(content) }}</a>
-    </h5>
-</div>
-```
+![Content Scheduler with scheduled content](img/enterprise_tut_choosing_airtime.png)
 
-and an entry in `app/config/views.yml` on the same level as `full` key:
+Publish the Page.
 
-``` yaml
-list:
-    article:
-        template: 'list/article.html.twig'
-        match:
-            Identifier\ContentType: 'article'
-```
-
-Like before, you must add one more image alias to the `app/config/image_variations.yml` file:
-
-``` yaml
-other_article:
-    reference: null
-    filters:
-        - {name: geometry/scaleheightdownonly, params: [120]}
-        - {name: geometry/crop, params: [120, 100, 0, 0]}
-```
-
-Finally, provide the new block with some styling. Add the following code at the end of the `web/assets/css/style.css` file:
-
-``` css
-/* Other articles schedule block */
-.other-articles-block {
-    padding-top: 20px;
-}
-
-.other-article {
-    clear: left;
-    padding-top: 5px;
-}
-
-.other-article-image {
-    float: left;
-    margin-right: 18px;
-}
-
-.other-article h5 {
-    padding-top: 25px;
-    font-size: 1.2em;
-}
-
-.other-article-link:link,
-.other-article-link:visited {
-    font-size: 1.2em;
-}
-```
-
-You are now able to add a new Schedule block to the Front Page and select the "List Schedule" block layout.
-Give the block an easily recognizable name, such as "Other Articles".
-Add two Articles to it to see how their look will differ from the featured ones.
-
-![The Page after adding a List Schedule Block](img/enterprise_tut_after_list_schedule_block.png "The Page after adding a List Schedule Block")
-
-### Set up overflow
-
-Now you will make use of the overflow functionality.
-
-In the settings of the Featured Articles block turn on overflow and select the Other Articles block as its overflow target.
+Now open the Timeline at the top of the screen.
+You can move the slider to different times and preview what the Content Scheduler block will look like at different hours.
+Content will be hidden if you moved to a point before it airs.
```

Comment:
Content will be shown when you move slider to a point when it airs. - preview shows current view and you have to move the slider to see the article